rpc-services/qubesbuilder.ProcessGithubCommand

```diff
@@ -16,7 +16,7 @@
 #
 # See README.md for detailed description.
 
-set -e
+set -eo pipefail
```

Comment:
For this, you need to change shebang to bash.

Reply:
Yeah, good point.